pkg/cloudprovider/cloud.go

```diff
@@ -23,6 +23,7 @@ import (
 
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/kubernetes/pkg/api/v1"
+	"k8s.io/kubernetes/pkg/client/informers/informers_generated/externalversions"
```

Comment:
You could prefix this with "informers" so that you can keep the same package name, similar to how it was done in controllermanager.go

Reply:
good to know :)